<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
<title>MOST - Metabolic Optimization and Simulation Tool</title>
<LINK href="main.css" rel="stylesheet" type="text/css">
<meta name="description" content="" />
<STYLE>
<!--
A{text-decoration:none}
-->
</STYLE>
</head>
<body bgcolor=ffffff text=000000 link=a0a0a0 vlink=a0a0ff alink=a0a000>

	<table align=center cellspacing="0" cellpadding="0"><tr height=15>
		<td align=center width=120><p><a href="index.html" name="top">Home</a></p></td>
		<td align=center width=120><p><a href="download.html">Download</a></p></td>
		<td align=center width=120><p><a href="publications.html">Publications</a></p></td>
		<td align=center width=120><p><a href="people.html">People</a></p></td>
		<td align=center width=120><p><a href="help.html">Help</a></p></td>
		<td align=center width=120><p><a href="tutorials.html">Tutorials</a></p></td>
	</table>

			<!-- Title -->
	
	<!-- Title -->
	
	<p align=center><font size=7><a href="index.html">MOST</a></font></p>
	<p align=center><font size=5>Metabolic Optimization and Simulation Tool</font></p>
	<hr>
	
	<table width=1000 cellspacing="1" cellpadding="1">
	<tr>
	<td width=140 valign=top bgcolor=a0a0a0 align=top>
	<p><font size=3>Help Topics</font></p>

	<p><font size=2><a href="help.html#contact">Contact</a></font></p>

	<p><font size=2><a href="help.html#about">About MOST</a></font></p>

	<p><font size=2><a href="help.html#overview">Overview </a></font></p>
	<ul><li><font size=2><a href="help.html#sort">Sorting Tables</a></font></li>
	<li><font size=2><a href="help.html#formula">Formula Bar</a></font></li>
	<li><font size=2><a href="help.html#rename">Renaming Metabolites</a></font></li></ul>

	<p><font size=2><a href="installing_most.html">Installing MOST</a></font></p>
	<ul><li><font size=2><a href="installing_most.html#req">Requirements</a></font></li>

	<li><font size=2><a href="installing_most.html#win">Windows</a></font></li>
	<ul><li><font size=2><a href="installing_most.html#issues">Known Issues</a></font></li></ul>

	<li><font size=2><a href="installing_most.html#linux">Linux</a></font></li>
	<li><font size=2><a href="installing_most.html#mac">Macintosh</a></font></li></ul>

	<p><font size=2><a href="file_menu.html">File Menu</a></font></p>

	<ul><li><font size=2><a href="file_menu.html#loadsbml">Load SBML</a></font></li>
	<li><font size=2><a href="file_menu.html#loadcsv">Load CSV</a></font></li>

	<ul><li><font size=2><a href="file_menu.html#mcol">Metabolites Column Selector</a></font></li>
	<li><font size=2><a href="file_menu.html#rcol">Reactions Column Selector</a></font></li>
	<li><font size=2><a href="file_menu.html#addmetab">Add Metabolite Prompt</a></font></li></ul>

	<li><font size=2><a href="file_menu.html#excel">Load Excel Spreadsheets</a></font></li>
	<li><font size=2><a href="file_menu.html#collection">Load from Model Collection</a></font></li>
	<li><font size=2><a href="file_menu.html#saving">Saving Files</a></font></li>

	<ul><li><font size=2><a href="file_menu.html#save">Save</a></font></li>
	<li><font size=2><a href="file_menu.html#savesbml">Save As SBML</a></font></li>
	<li><font size=2><a href="file_menu.html#savecsv">Save As CSV</a></font></li></ul>

	<li><font size=2><a href="file_menu.html#clear">Clear Tables</a></font></li>
	<li><font size=2><a href="file_menu.html#exit">Exit</a></font></li></ul>

	<p><font size=2>Analysis Menu</font></p>

	<ul><li><font size=2><a href="#fba">FBA</a></font></li>
	<li><font size=2><a href="#gdbb">GDBB</a></font></li>
	<li><font size=2><a href="#eflux2">E-Flux2</a></font></li>
	<li><font size=2><a href="#spot">SPOT</a></font></li></ul>

	<p><font size=2><a href="edit_menu.html">Edit Menu</a></font></p>

	<ul><li><font size=2><a href="edit_menu.html#unused">Unused Metabolites</a></font></li>
	<li><font size=2><a href="edit_menu.html#suspicious">Suspicious Metabolites</a></font></li>
	<li><font size=2><a href="edit_menu.html#undo">Undo/Redo</a></font></li>
	<li><font size=2><a href="edit_menu.html#find">Find/Replace</a></font></li>
	<li><font size=2><a href="edit_menu.html#select">Select All</a></font></li>
	<li><font size=2><a href="edit_menu.html#add">Add Row/Column</a></font></li>
	<li><font size=2><a href="edit_menu.html#unsort">Unsort</a></font></li></ul>

	<p><font size=2><a href="options_menu.html">Options Menu</a></font></p>

	<ul><li><font size=2><a href="select_solvers.html">Select Solvers</a></font></li>
	<li><font size=2><a href="solver_parameters.html">Set Solver Parameters</a></font></li></ul>

	<p><font size=2><a href="toolbar.html">Toolbar</a></font></p>

	<ul><li><font size=2><a href="toolbar.html#save">Save</a></font></li>
	<li><font size=2><a href="toolbar.html#copy">Copy/Paste</a></font></li>
	<li><font size=2><a href="toolbar.html#undo">Undo/Redo</a></font></li>
	<li><font size=2><a href="toolbar.html#find">Find/Replace</a></font></li></ul>

	<p><font size=2><a href="context_menu.html">Context Menus</a></font></p>

	<ul><li><font size=2><a href="context_menu.html#table">Table Menus</a></font></li>
	<ul><li><font size=2><a href="context_menu.html#select">Select Row(s)/Column(s)/Cell(s)</a></font></li>
	<li><font size=2><a href="context_menu.html#selectall">Select All</a></font></li>
	<li><font size=2><a href="context_menu.html#copy">Copy/Paste</a></font></li>
	<li><font size=2><a href="context_menu.html#clear">Clear Contents</a></font></li>
	<li><font size=2><a href="context_menu.html#delrow">Delete Row(s)</a></font></li>
	<li><font size=2><a href="context_menu.html#highpar">Highlight Participating Reactions</a></font></li>
	<li><font size=2><a href="context_menu.html#unhighpar">Un-Highlight Participating Reactions</a></font></li>
	<li><font size=2><a href="context_menu.html#reac">Reaction Editor</a></font></li></ul>
	<li><font size=2><a href="context_menu.html#header">Table Header Menus</a></font></li>
	<ul><li><font size=2><a href="context_menu.html#delcol">Delete Column</a></font></li></ul>
	<li><font size=2><a href="context_menu.html#formula">Formula Bar Menu</a></font></li>
	<li><font size=2><a href="context_menu.html#console">Console Menu</a></font></li>
	<ul><li><font size=2><a href="context_menu.html#popout">Console Popout</a></font></li></ul>
	<li><font size=2><a href="context_menu.html#tree">Tree Menu</a></font></li></ul>

	<p><font size=2><a href="keyboard_shortcuts.html">Keyboard Shortcuts</a></font></p>
	<ul><li><font size=2><a href="keyboard_shortcuts.html#shortcuts">List of Keyboard Shortcuts</a></font></li></ul>

	</td>

	<td width=600 valign=top>
	<p align=center><font size=5>Analysis Menu</font></p>

	<p>Note: Results of Analyses are not editable. After running an Analysis, the result will be selected in the tree (panel on right side of MOST).
	When a result is selected, any menu item that could be used to change the results will be grayed out. This includes most of the menu items
	in the <a href="edit_menu.html">Edit</a> menu and <a href="context_menu.html">Context</a> menus.</p>

	<p><a name="fba"><strong>  FBA:</strong></a></p>

	<blockquote>Before running FBA, the Biological Objective for a reaction should be set to 1.0. Usually the biomass reaction is chosen
	as the Biological Objective to be maximized. Drain metabolites (usually abbreviations end with "_b" in SBML files) should have
	Boundary Condition set to true. If these are set wrong, an "Infeasible Model" error will be displayed when FBA is run.
	All of the models in the <a href="file_menu.html#collection">Model Collection Database</a>, and SBML models
	from <a href="http://blog.theseed.org/model_seed/">Model Seed</a> have the Biological Objective of biomass 
	reaction set to 1.0 and Boundary Conditions set correctly.</blockquote>

	<blockquote>FBA has an option to include FVA (Flux Variability Analysis) results.Running FBA with FVA
	results takes significantly more time than running FBA without FVA.</blockquote>

	<blockquote>Select Analysis –> FBA (Alt+A then Alt+F).</blockquote>

	<blockquote>The FVA Dialog will appear:</blockquote>

	<p align="center"><img src ="images/fva_analysis.jpg" alt="Image - FVA Dialog" border="0"></p>

	<blockquote>Selecting the "Run Euclidean norm minimization" checkbox finds a unique flux set by minimizing the length of the 
	flux vector while maintaining the FBA Biomass flux (effectively, it is <a href="#eflux2">E-Flux2</a> 
	without formatting the reactions with gene expression data). This option provides a unique flux distribution.</blockquote>

	<blockquote>Click "Yes" to include FVA results, "No" to just run FBA.</blockquote>

	<blockquote>After FBA is finished, a link to the results table will be added to the tree, and results will be displayed in the console.</blockquote>

	<blockquote>If FVA was selected, two new columns, Min. Flux. and Max. Flux, will be added to the Reactions Table.</blockquote>

	<p align="center"><a href="images/fva_done.jpg" target="_blank">
	<img src ="images/fva_done.jpg" alt="Image - FVA Results" border="0" height="90%" width="90%"></a></p>

	<blockquote>Otherwise, MOST will appear as normal.</blockquote>

	<p align="center"><a href="images/fba_done.jpg" target="_blank">
	<img src ="images/fba_done.jpg" alt="Image - FBA Results" border="0" height="90%" width="90%"></a></p>

	<br>

	<blockquote>If you are using Gurobi as a solver and your Gurobi license is not valid, the following error will appear:</blockquote>
	
	<p align="center"><img src ="images/no_license.jpg" alt="Image - No License" border="0"></p>

	<blockquote>Your Gurobi license needs to be updated or refreshed.</blockquote>

	<br>

	<blockquote>The following article explains Flux Balance Analysis:</blockquote>

	<blockquote>J. Orth, I. Thiele, B. Palsson, <a href="http://dx.doi.org/10.1038/nbt.1614">What is flux balance analysis?</a>, 
	Nature Biotechnology 28, 245–248 (2010).</blockquote>

	<blockquote>The following article explains Flux Variability Analysis:</blockquote>

	<blockquote>R. Mahadevan, C.H. Schilling, <a href="http://dx.doi.org/10.1016/j.ymben.2003.09.002">
	The effects of alternate optimal solutions in constraint-based genome-scale metabolic models</a>, 
	Metabolic Engineering 5, 264–276 (2003).</blockquote>

	<br>
	<p><a name="gdbb"><strong>  GDBB:</strong></a></p>

	<blockquote>Before running GDBB, the Biological Objective for a reaction should be set to 1.0. Usually the biomass reaction is chosen
	as the Biological Objective to be maximized.
	Also a Synthetic Objective should be set for the substance the user wishes to maximize. 
	Synthetic objective is the vector <i>g</i> in Equation 2 from
	<a href="#Egen">D. Egen and D. S. Lun, 2012</a> and Equation 3 from <a href="#Lun">D. S. Lun et al., 2009</a>.
	For example if the user wishes to 
	maximize acetate production, the Synthetic Objective of the Acetate Exchange reaction should be set to 1.0.
	The models Ec_iAF1260_anaerobic_glc10_acetate and Ec_iAF1260_anaerobic_glc10_succinate
	in the <a href="file_menu.html#collection">Model Collection Database</a> have
	the Biological and Synthetic Objectives set correctly for maximizing acetate and succinate production respectively.</blockquote>

	<blockquote>Select Analysis –> GDBB (Alt+A then Alt+G). The GDBB dialog will appear:</blockquote>

	<p align="center"><img src ="images/gdbb_dialog2.jpg" alt="Image - GDBB Dialog" border="0"></p>

	<blockquote>The buttons, check boxes and text fields on the GDBB dialog can all be accessed using 
	keyboard shortcuts by using Alt+ and underlined letter. </blockquote>

	<blockquote>Example:  Number of Knockouts (Alt+ K)</blockquote>

	<blockquote>Click the Start buton (Alt+S) to start GDBB running.</blockquote>

	<blockquote>If Finite Optimizer Time Limit was selected and time has run out, the Stop button was clicked, or the frame close ("X") 
	button was clicked, any optimizations that have been initialized will still need to complete, so the Time Elapsed
	label will change to Processing . . until all optimizations are complete.</blockquote>

	<blockquote>After GDBB is finished, the GDBB dialog will close.</blockquote>

	<blockquote>After GDBB is finished, a folder will be added to the tree containing all solutions generated by the analysis, 
	and results will be displayed in the console.</blockquote>

	<br>

	<blockquote>If you are using Gurobi as a solver and your Gurobi license is not valid, the following error will appear:</blockquote>
	
	<p align="center"><img src ="images/no_license.jpg" alt="Image - No License" border="0"></p>

	<blockquote>Your Gurobi license needs to be updated or refreshed.</blockquote>

	<br>
	<p><a name="eflux2"><strong>  E-Flux2:</strong></a></p> 

	<blockquote>E-Flux2 is a method for integration of transcriptomic data into constraint-based models when there is 
	a known objective function <a href="#Kim">(Kim et al. Submitted)</a>.</blockquote>

	<blockquote>Before running E-Flux2, the Biological Objective for a reaction should be set to 1.0. Usually the biomass reaction is chosen
	as the Biological Objective to be maximized.
	All of the models in the <a href="file_menu.html#collection">Model Collection Database</a>, and SBML models
	from <a href="http://blog.theseed.org/model_seed/">Model Seed</a> have the Biological Objective of biomass 
	reaction set to 1.0.</blockquote>

	<blockquote>E-Flux2 requires a CSV file with two columns where the first column in the gene name and the
	second column is the gene expression. See example below for expected file format:</blockquote>

	<p align="center"><img src ="images/gene_expr.jpg" alt="Image - Gene Expression" border="0"></p>

	<blockquote>Sample gene expression data for Escherichia coli K-12 MG1655 is available and can be downloaded <a href="gene_expr.csv" target="_blank">here</a>. 
	This file can be used for running E-Flux2 on all Escherichia coli K-12 MG1655 models in the <a href="file_menu.html#collection">Model Collection</a>.</blockquote>

	<blockquote>Select Analysis –> E-Flux2 (Alt+A then Alt+E).</blockquote>

	<blockquote>A file browser will appear. Select a gene expression csv file, then click the Open button
	on the file browser.</blockquote>

	<p align="center"><a href="images/load_gene_expr.jpg" target="_blank">
	<img src ="images/load_gene_expr.jpg" alt="Image - Load Gene Expression" border="0" height="90%" width="90%"></a></p>

	<blockquote>If none of the gene names in the gene expression file match the names of genes in the currently loaded model, an
	 error message will appear. The gene expression file that has been loaded is most likely not the correct file for that model.
	</blockquote>

	<p align="center"><a href="images/no_matches.jpg" target="_blank">
	<img src ="images/no_matches.jpg" alt="Image - No Matches" border="0" height="90%" width="90%"></a></p>

	<blockquote>Otherwise, a progress bar will appear:</blockquote>

	<p align="center"><a href="images/eflux2_progress.jpg" target="_blank">
	<img src ="images/eflux2_progress.jpg" alt="Image - E-Flux2 progress" border="0" height="90%" width="90%"></a></p>

	<blockquote>After E-Flux2 is finished, a link will be added to the tree, and results will be displayed in the console.</blockquote>

	<p align="center"><a href="images/eflux2_done.jpg" target="_blank">
	<img src ="images/eflux2_done.jpg" alt="Image - E-Flux2 Results" border="0" height="90%" width="90%"></a></p>

	<br>
	<p><a name="spot"><strong> SPOT</strong> (Simplified Pearson cOrrelation with Transcriptomic data):</a></p> 

	<blockquote>SPOT is a method for integration of transcriptomic data into constraint-based models when there is 
	no known objective function <a href="#Kim">(Kim et al. Submitted)</a>.</blockquote>

	<blockquote>SPOT requires a CSV file with two columns where the first column in the gene name and the
	second column is the gene expression. See example below for expected file format:</blockquote>

	<p align="center"><img src ="images/gene_expr.jpg" alt="Image - Gene Expression" border="0"></p>

	<blockquote>Sample gene expression data for Escherichia coli K-12 MG1655 is available and can be downloaded <a href="gene_expr.csv" target="_blank">here</a>. 
	This file can be used for running SPOT on all Escherichia coli K-12 MG1655 models in the <a href="file_menu.html#collection">Model Collection</a>.</blockquote>

	<blockquote>Select Analysis –> SPOT (Alt+A then Alt+S).</blockquote>

	<blockquote>A file browser will appear. Select a gene expression csv file, then click the Open button
	on the file browser.</blockquote>

	<p align="center"><a href="images/load_gene_expr.jpg" target="_blank">
	<img src ="images/load_gene_expr.jpg" alt="Image - Load Gene Expression" border="0" height="90%" width="90%"></a></p>

	<blockquote>If none of the gene names in the gene expression file match the names of genes in the currently loaded model, an
	 error message will appear. The gene expression file that has been loaded is most likely not the correct file for that model.
	</blockquote>

	<p align="center"><a href="images/no_matches.jpg" target="_blank">
	<img src ="images/no_matches.jpg" alt="Image - No Matches" border="0" height="90%" width="90%"></a></p>

	<blockquote>Otherwise, a progress bar will appear.</blockquote>

	<p align="center"><a href="images/spot_progress.jpg" target="_blank">
	<img src ="images/spot_progress.jpg" alt="Image - SPOT progress" border="0" height="90%" width="90%"></a></p>

	<blockquote>After SPOT is finished, a link will be added to the tree, and results will be displayed in the console.</blockquote>

	<p align="center"><a href="images/spot_done.jpg" target="_blank">
	<img src ="images/spot_done.jpg" alt="Image - SPOT Results" border="0" height="90%" width="90%"></a></p>

	<br>

	<blockquote>References</blockquote>
	<blockquote><a name="Egen">D. Egen</a> and D. S. Lun. <A HREF="http://dx.doi.org/10.1093/bioinformatics/bts255">Truncated branch and 
	bound achieves efficient constraint-based genetic design</A>.  <EM>Bioinformatics</EM>, 28(12):1619-1623, June 2012.</blockquote>

	<blockquote><a name="Kim">Kim</a>, M. et al. Integration of transcriptomic data in genome-scale metabolic models predicts measured 
	intracellular central carbon metabolic fluxes with high correla-tion in Escherichia coli and Saccharomyces cerevisiae. Submitted.</blockquote>

	<blockquote><a name="Lun">D. S. Lun,</a> G. Rockwell, N. J. Guido, M. Baym, J. A. Kelner, B. Berger, J. E. Galagan, and G. M. Church. 
	<A HREF="http://dx.doi.org/10.1038/msb.2009.57">Large-scale identification of genetic design strategies using local search</A>. 
	<EM>Mol. Syst. Biol.</EM>, 5:296, August 2009.</blockquote>

	</td>
	</tr>
	</table>

</body>
</html>